TechDebtsTracking/core/src/main/java/com/novoda/support/annotations/TechDebt.java

```diff
@@ -0,0 +1,32 @@
+package com.novoda.support.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.SOURCE)
+@Target({ElementType.ANNOTATION_TYPE,
+        ElementType.CONSTRUCTOR,
+        ElementType.FIELD,
+        ElementType.LOCAL_VARIABLE,
+        ElementType.METHOD,
+        ElementType.PACKAGE,
+        ElementType.PARAMETER,
+        ElementType.TYPE})
+public @interface TechDebt {
+
+    enum Size {
+        XSMALL,
```

Comment:
out of curiosity - what's the rationale of introducing this dimension? how would you decide the size of tech debt?